tests/dragonfly/acl_family_test.py

```diff
@@ -220,27 +220,31 @@ async def test_acl_cat_commands_multi_exec_squash(df_factory):
     await client.close()
 
 
-@pytest.mark.skip("Skip because it fails on arm release")
 @pytest.mark.asyncio
 async def test_acl_deluser(df_server):
     client = aioredis.Redis(port=df_server.port)
 
-    res = await client.execute_command("ACL SETUSER george ON >pass +@transaction +@string")
-    assert res == b"OK"
-
-    res = await client.execute_command("AUTH george pass")
-    assert res == b"OK"
+    assert (
+        await client.execute_command("ACL SETUSER george ON >pass +@transaction +set ~*") == b"OK"
```

Comment:
Also not that I refactored the code slightly, `df_server.client()` above and remove the "b" it's noise :D

Reply:
if it currently passes, it's not a noise. The right fix is use `async_client` fixture that is already configured with `decode_responses`.

Reply:
@romange read my comment. I said use `df_server.client()` which uses `decode_responses` internally. If he does that (or use async_client -- it doesn't matter) he can remove the `"b"` which is noise

Reply:
And I also already pushed a refactor for this